layouts/_default/single.html

```diff
@@ -1,18 +1,12 @@
 {{ define "main" }}
-  {{ if eq .Type "blog" }}
```

Comment:
This change causes the date to appear in 'pages' that should not have a date. The date shows as '01-01-0001'.